src/compiler/diagnosticMessages.json

```diff
@@ -3016,7 +3016,10 @@
         "category": "Error",
         "code": 2792
     },
-
+    "Namespace '{0}' has no exported member '{1}'. Did you mean '{2}'?": {

```

Comment:
Is there already a diagnostic for modules? Maybe just reuse that and remove the word "module"

Reply:
https://github.com/microsoft/TypeScript/blob/10fb9c9381e83201450adc97b7298322d3711370/src/compiler/diagnosticMessages.json#L2755-L2757

Do you mean to change this message to 

`'{0}' has no exported member '{1}'. Did you mean '{2}'?`?

Reply:
I just tried to follow the original message 

https://github.com/microsoft/TypeScript/blob/10fb9c9381e83201450adc97b7298322d3711370/src/compiler/diagnosticMessages.json#L2634-L2636

Reply:
Yes, exactly - consider switching to

```
'{0}' has no exported member '{1}'. Did you mean '{2}'?
```